incubator/kubeless/values.yaml

```diff
@@ -10,7 +10,7 @@ controller:
     functionController:
       image:
         repository: kubeless/function-controller
-        tag: v1.0.4
+        tag: v1.0.5
```

Comment:
Can you update the versions of the http and cronjob trigger controllers as well? (The latest ones are `v1.0.1`)

Reply:
I noticed that the v1.0.1 tags aren't available in the [bitnami namespace](https://hub.docker.com/r/bitnami/http-trigger-controller/tags) in dockerhub that's used in this chart. It's present in the [kubeless namespace](https://hub.docker.com/r/kubeless/http-trigger-controller/tags) however, should I switch the images to that?

Reply:
ah, I didn't notice that these were bitnami images. Yes. please, use the `kubeless` namespace instead.

Reply:
Ok @andresmgot , that's been updated.